src/App.tsx

```diff
@@ -71,8 +72,22 @@ import campusData from './assets/campuses.json';
 const TOKEN = import.meta.env.VITE_MAPBOX_TOKEN;
 const FOOTER_MENU_ID = import.meta.env.VITE_REMOTE_FOOTER_MENU_ID;
 const FOOTER_SOCIAL_ID = import.meta.env.VITE_REMOTE_SOCIAL_LINKS_ID;
+const REACT_MEASUREMENT_ID = import.meta.env.VITE_REACTGA_MEASUREMENT_ID;
 
 function App() {
+  ReactGA.initialize(REACT_MEASUREMENT_ID, {
+    gaOptions: {
+      debug_mode: true,
+    },
+    gtagOptions: {
+      debug_mode: true,
+    },
+  });
+
+  // Send pageview with a custom path
+  ReactGA.send({ hitType: "pageview", page: "/", title: "UCF Campus Map" });
```

Comment:
We need to see what happens when the measurement ID isn't set and make sure the app still runs without it.

Reply:
Conditional state added here.